### DISCUSSION TITLE
Exporting a PDF issue

### DISCUSSION THREAD

**Discussion by geoff-maddock** [davcd/jsonresume-theme-actual#18] [Q&A] [answered]
Pulled this down to try it out - the HTML export works great.

```
cutups@Hydra-PC:/mnt/c/Users/cutups/code/projects/geoff-maddock.github.io$ resume export resume.html --format html --theme actual

Done! Find your new .html resume at:
 /mnt/c/Users/cutups/code/projects/geoff-maddock.github.io/resume.html
```

However, when I try to export pdf, it throws an error that doesn't quite make sense to me, as the theme is installed (I just used it above).
```
cutups@Hydra-PC:/mnt/c/Users/cutups/code/projects/geoff-maddock.github.io$ resume export resume.pdf --format pdf --theme actual
You have to install this theme relative to the folder to use it e.g. `npm install jsonresume-theme-actual`
```

Any tips?  I'd love to be able to do a PDF export.

**Comment by geoff-maddock** [comment, marked as answer]
Looking into this a bit more...clearly this is an issue with the resume library, so not this theme at all.
Going down the path here looking into how to get it to work:
https://github.com/jsonresume/resume-cli/issues/564

**Comment by davcd** [comment]
Hello @geoff-maddock, many thanks for your comments!
As mentioned, this is an issue related with the resume-cli library.
As a workaround you can install the dependency in both local and global context and it should work.

**Comment by davcd** [comment]
I just created https://github.com/jsonresume/resume-cli/issues/673.

**Reply by davcd** [reply]
Now as this PR was accepted, the steps defined in readme should be fine!

``` bash
npm install jsonresume-theme-actual

resume export resume.pdf --format pdf --theme actual
resume export resume.html --format html --theme actual
```
